src/suricata.c

```diff
@@ -691,8 +691,10 @@ static void PrintBuildInfo(void)
 #ifdef HAVE_HTP_URI_NORMALIZE_HOOK
     strlcat(features, "HAVE_HTP_URI_NORMALIZE_HOOK ", sizeof(features));
 #endif
-#ifdef PCRE_HAVE_JIT
+#ifdef PCRE2_HAVE_JIT
+    // TODOpcre2ask keep for backwards compatibility ?
```

Comment:
I don't see it used in SV tests, so not sure what the compatibility issue would be. OTOH we can just keep it as is (not adding PCRE2_JIT) because we'll just have one pcre in use, right?

Reply:
Indeed, I think we can just keep `PCRE_JIT` in the features and not add `PCRE2_JIT`